### DISCUSSION TITLE
Linking Agent Name used to a file's physical O/S-based name

### DISCUSSION THREAD

**Discussion by ms52538** [CodyBerenson/PGMA-Modernized#311] [General]
Is it possible to somehow indicate, in the scene/feature file's physical name, the agent named used to make the match, without throwing off the agent matching and scraping? I know it is populated in a scene/feature matching page. I'm more interested in having this information at the physical file level as it would help me to track back to what source I used to name the file.  In other words: it's great when there's a match, it sucks where the match goes bad, or fails to match all together. I guess some sort of nullified symbols that can be used at the end of the file name, before the extension, that would be ignored by the agents scanning. As I already have to go in and manually change a file's name so it can match, might as well add the source used to match the file.

**Comment by JPH71** [comment]
That would mean renaming the physical file..
So would not be appropriate..

**Comment by ms52538** [comment]
The question is: can the agents be coded in a way that can ignore portions of a file's name (presumably surrounded with a couple of special characters)? The theory would be enable an optional feature for those who want the added benefit for whatever purpose they may see fit.
